tardis_env3.yml

```diff
@@ -74,3 +73,5 @@ dependencies:
 
   # Other
   - git-lfs
+  - pip: # see https://github.com/h5py/h5py/issues/1880
+    - h5py
```

Comment:
What relies on h5py? Adding more dependencies is always a risk.

Reply:
We agreed a long time ago not to use pip dependencies. 

That potentially breaks conda-forge package and will break the continuous delivery of TARDIS since `conda-lock` ignores pip dependencies.

If you need the pip package for this pipeline, you can do `pip install h5py` in a new step and overwrite the conda package.

Reply:
`asv` creates its own environment every time, I'm afraid I'd have to look for an alternate solution.

Reply:
I don't understand how ASV works, but I can help I think.

Reply:
Thank you so much!